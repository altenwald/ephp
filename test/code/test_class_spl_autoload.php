<?php

function __autoload($classname) {
    print "including... $classname\n";
    require_once(__DIR__ . "/test_$classname.php");
}

function my_autoload($classname) {
    print "not found in __autoload: $classname\n";
}

spl_autoload_register("__autoload");
spl_autoload_register("my_autoload", false, true);
print "BEGIN\n";
spl_autoload_call("myclass");
print "LOADED\n";
#$a = new myclass;
#var_dump($a);
