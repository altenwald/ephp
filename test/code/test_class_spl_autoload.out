BEGIN
not found in __autoload: myclass
including... myclass
LOADED
